evennia/web/webclient/static/webclient/js/evennia.js

```diff
@@ -50,6 +50,7 @@ An "emitter" object must have a function
 */
 
 (function() {
+    "use strict"
```

Comment:
I was not aware of "use strict", it makes sense to use it. Out of curiosity, my understanding is that "use strict" can be used at the file level rather than the function level. Is there some reason not to put it on the file level?